index.js

```diff
@@ -155,21 +157,6 @@ function randomCase(input) {
   return convert(input, () => Math.round(Math.random()));
 }
 
-/**
- * Converts matched letters to a specified case
- * @param {string} input - string to convert
- * @param {(string | RegExp)} regex - use to match letters you want to convert
- * @param {function} caseReplacer - a function that changes the letter's case
- * @return {string} given input with match letters to the  case type determined
- */
-function convertToSpecificCase(input, regex, caseReplacer) {
```

Comment:
Why is this function removed?! It is used at https://github.com/strdr4605/mockingcase/blob/master/src/mockingcase.js#L54

Reply:
Very sorry @strdr4605 . I will add it and give you a pull request. Is that the only error?

Reply:
Till now no, but I will check again after the update. Please remove `.idea` folder, and `rebase` your branch on top of the main master branch

Reply:
Okay. I will rebase my branch and give you a PR within a day.